scaleway/data_source_instance_volume_test.go

```diff
@@ -1,21 +1,33 @@
 package scaleway
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
 func TestAccScalewayDataSourceInstanceVolume_Basic(t *testing.T) {
+	tt := NewTestTools(t)
+	defer tt.Cleanup()
+	volumeName := newRandomName("volume")
```

Comment:
This type of randomness could cause problems with cassette. We should consider the generation a name based on the test name. Or remove uniqness if not needed